addons/payment_paypal/models/payment.py

```diff
@@ -25,6 +25,7 @@ class AcquirerPaypal(models.Model):
         'Paypal Merchant ID', groups='base.group_user',
         help='The Merchant ID is used to ensure communications coming from Paypal are valid and secured.')
     paypal_use_ipn = fields.Boolean('Use IPN', default=True, help='Paypal Instant Payment Notification', groups='base.group_user')
+    pdt_token = fields.Char(string='PDT Token', required_if_provider='paypal', help='Payment Data Transfer allows you to receive notification of successful payments as they are made.', groups='base.group_user')
```

Comment:
Please respect the payment providers convention: all fields should be prefixed by the provider name.